src/formatGeneratedModule.ts

```diff
@@ -2,6 +2,36 @@ import { FormatModule } from "relay-compiler";
 import * as ts from "typescript";
 import addAnyTypeCast from "./addAnyTypeCast";
 
+const createRequireRegex = () => /require\("(.*)"\)/g;
```

Comment:
@n1ru4l The regex used to replace require calls is off. In my case, the paths of the required modules are delimited by single quotes, whereas the regex looks for double quotes.

E.g.: The regex will not identify the require call in the following line: `"operation": require('./BlogPaginationQuery.graphql.ts')`

After replacing the double quotes in the regex by single quotes, require calls were successfully replaced for me.

Reply:
@asterikx you could help pushing this getting merged by creating a pr that adds tests to this branch

Reply:
@n1ru4l Can you have a look at https://github.com/n1ru4l/relay-compiler-language-typescript/pull/1? If I can help any further to get this merged, let me know.

Reply:
@asterikx I did not see the PR, awesome! I will check it out next week!